### DISCUSSION TITLE
Left-hand side menu with site contents

### DISCUSSION THREAD

**Discussion by olayway** [flowershow/flowershow#332] [General and Help]
Some users may want to show a left-hand side sidebar listing all their site contents. This makes sense especially for users with very large vaults.

**Comment by danymat** [comment]
I Will add my two cents
Hello!

I'm right now with a vault of ~600 notes, following ZK philosophy (https://danielmathiot.tech/notes)

I will surely migrate in the next few days, and I have a lot of insights after using obsidian publish.

Nevertheless, related to this discussion:

- a toggle option to show/not show the notes in the left in the website 
- a button to display/hide the sidebar
- some customization option to put it at left/right of the page
- having a list of notes to always show at the left 
  - indeed, some options to automatically add files (by regex, tag) or a manual notes selection 

I hope that arises some interesting discussion, feel free to ask me more questions if I didn't express it good enough

Thanks again for all your work, I'm surely gonna contribute as soon as I can.

**Reply by rufuspollock** [reply]
@danymat thanks for the valuable reflections.

**Comment by rufuspollock** [comment]
@olayway i think this is the same kind of topic as covered in https://github.com/flowershow/flowershow/issues/94

I do like the tailwindcss.com/docs approach - though it is for a docs site it applies i think similarly to a pure digital garden. This obviously can't show all notes on a largish site so it probably has to be customizable at some point.

![](https://user-images.githubusercontent.com/180658/159502659-2833ce95-b55b-4333-b34b-94b7a8284c3a.png)

**Comment by StefanoCecere** [comment]
my needs is that i want to use flowerapp for any kind of website, with a fully configurable left side menu.
following folders structures and metadata (also with weight metafield to have manual sorting of entries)

MkDocs manages this very easily.. this is a site a created from a simple Obsidian vault (github repo linked)

<img width="783" alt="Screenshot 2022-12-04 at 17 18 50" src="https://user-images.githubusercontent.com/640536/205502487-35bca819-1dac-4457-bd66-73e60f9da46e.png">

there we can configure two options:
1) the top folders get in the horizontal top menu, and sub to the left
2) the top folders are in the left menu, with hierarchies

i very much prefer a configurable horizontal menu.. maybe "mapped" to some top level folder
